triton/trt/README.md

```diff
@@ -0,0 +1,57 @@
+# Best Practice of TensorRT acceleration for K2 models
```

Comment:
Could you move the doc to
https://github.com/k2-fsa/sherpa/tree/master/docs/source/triton

You can place a link to the doc in this README.md


---

The generated doc can be viewed at
https://k2-fsa.github.io/sherpa/triton/overview.html

---

https://github.com/k2-fsa/sherpa/blob/master/docs/README.md
describes how to build the doc locally.

Reply:
Will do.

Reply:
Hi Fangjun, I've added docs into sherpa/docs. Please check. Thanks